privly.safariextension/scripts/background_scripts/first_run.js

```diff
@@ -61,14 +57,21 @@ var firstRun = {
   checkFirstRun: function() {
 
     // Set the expected version to compare against the stored version
-    // todo, set this dynamically
-    var runningVersion = "0.1.0";
-    var lastRunVersion = ls.getItem("version");
+    // Set the expected version dynamically if possible
+
+    var runningVersion;
+    /* istanbul ignore if */
+    if (typeof safari !== "undefined" && safari.extension !== undefined) {
+      runningVersion = safari.extension.displayVersion;
+    } else {
+      runningVersion = "0.1.0";
```

Comment:
Yes, this is called when the test is run.
In the scope of the JS test, the safari variable is not defined.